Automattic/jetpack
# Add/stats page upsells actions

This is the second step in the Upgrade Nudges on the Stats page.

This PR adds the button actions and tracking.

#### Changes proposed in this Pull Request:

* Add links to the buttons
* Tracks the events when buttons are clicked

#### Jetpack product discussion


p1HpG7-cfa-p2

#### Does this pull request change what data or activity we track or use?


We track the events when the new Upgrade Buttons on the Stats page are clicked

#### Testing instructions:






We are going to test 2 things: Tracking and actions

For all tests:

* Connect your site
* Edit your `wp-config.php` and add `define( 'JETPACK_DEV_TEST_STATS_UPGRADE_NUDGES', true );`
* Got to Jetpack > Site Stats

Testing tracking:

* You should see all 7 items on the bottom of the page. Click all of them and check Tracks live view for the following events: `jetpack_stats_nudges_security`, `jetpack_stats_nudges_scan`, `jetpack_stats_nudges_backup`, `jetpack_stats_nudges_akismet`, `jetpack_stats_nudges_search`, `jetpack_stats_nudges_crm`, and`jetpack_stats_nudges_boost`.

Testing the actions:

You'll need to test all the actions below with a site fully connected and then with a site connected only at a site level.

Security -> should take you to the Plans page. Make the purchase and confirm the plan is added to your site and you are redirected back to WP-Admin

Backup -> should take you to the Backup landing page. Make the purchase and confirm the plan is added to your site and you are redirected back to WP-Admin

Scan -> should take you to the Scan landing page. Make the purchase and confirm the plan is added to your site and you are redirected back to WP-Admin

Anti-spam -> should take you to the Anti-spam landing page. Make the purchase and confirm the plan is added to your site and you are redirected back to WP-Admin

Search -> should take you to the Search landing page. Make the purchase and confirm the plan is added to your site and you are redirected back to WP-Admin (NOTE: Search will not work for a site-only connection. We either have to make it work or hide this nudge. See p1626445219079300-slack-CBG1CP4EN
